packages/components/CHANGELOG.md

```diff
@@ -44,6 +43,7 @@
 ### Bug Fixes
 
 -   `ResizableBox`: Make drag handles focusable ([#67305](https://github.com/WordPress/gutenberg/pull/67305)).
+-   `CustomSelectControl`: Update correctly when `showSelectedHint` is enabled ([#67733](https://github.com/WordPress/gutenberg/pull/67733)).
```

Comment:
Tweaked the text a bit so it describes the change to any consumer of this component, not a user of the Gutenberg app.